Tool linker errors on Linux x86_64

I was doing a full Gradle build and got the following linker errors:
```
> Task :tools:datalogtool:linkDatalogtoolLinuxx86-64ReleaseExecutable FAILED
mold: error: undefined symbol: wpi::fields::GetFields()
>>> referenced by Field2D.cpp
>>>               /home/tav/frc/wpilib/allwpilib/glass/build/libs/glass/static/linuxx86-64/release/libglass.a(Field2D.o):(wpi::glass::DisplayField2DSettings(wpi::glass::Field2DModel*))
>>> referenced by Field2D.cpp
>>>               /home/tav/frc/wpilib/allwpilib/glass/build/libs/glass/static/linuxx86-64/release/libglass.a(Field2D.o):(wpi::glass::DisplayField2D(wpi::glass::Field2DModel*, ImVec2 const&))
collect2: error: ld returned 1 exit status

> Task :tools:sysid:linkSysidLinuxx86-64ReleaseExecutable FAILED
mold: error: undefined symbol: wpi::fields::GetFields()
>>> referenced by Field2D.cpp
>>>               /home/tav/frc/wpilib/allwpilib/glass/build/libs/glass/static/linuxx86-64/release/libglass.a(Field2D.o):(wpi::glass::DisplayField2DSettings(wpi::glass::Field2DModel*))
>>> referenced by Field2D.cpp
>>>               /home/tav/frc/wpilib/allwpilib/glass/build/libs/glass/static/linuxx86-64/release/libglass.a(Field2D.o):(wpi::glass::DisplayField2D(wpi::glass::Field2DModel*, ImVec2 const&))
collect2: error: ld returned 1 exit status
```
I assume a dependency is missing in the Gradle config, since that was the cause the last time an error like this happened.

--- a/tools/datalogtool/build.gradle
+++ b/tools/datalogtool/build.gradle
@@ -99,6 +99,7 @@ model {
                 }
                 it.cppCompiler.define("LIBSSH_STATIC")
                 lib project: ':glass', library: 'glass', linkage: 'static'
+                lib project: ':fields', library: 'fields', linkage: 'static'
                 lib project: ':wpigui', library: 'wpigui', linkage: 'static'
                 lib project: ':datalog', library: 'datalog', linkage: 'static'
                 lib project: ':thirdparty:imgui_suite', library: 'imguiSuite', linkage: 'static'

--- a/tools/sysid/build.gradle
+++ b/tools/sysid/build.gradle
@@ -98,6 +98,7 @@ model {
                     return
                 }
                 lib project: ':glass', library: 'glass', linkage: 'static'
+                lib project: ':fields', library: 'fields', linkage: 'static'
                 lib project: ':wpimath', library: 'wpimath', linkage: 'static'
                 lib project: ':wpigui', library: 'wpigui', linkage: 'static'
                 lib project: ':datalog', library: 'datalog', linkage: 'static'

